lib/oban/testing.ex

```diff
@@ -166,6 +169,13 @@ defmodule Oban.Testing do
     |> repo.one()
   end
 
+  defp available_jobs(repo) do
+    base_query([])
+    |> select([:worker, :args])
```

Comment:
Technically any field may be used to construct the query and that should be reflected in the inspected value. For example, if they were looking for `worker: MyWorker, attempt: 2, max_attempts: 3` all of those fields would be considered in the search but wouldn't be reflected here.

You can pull the keys out of `opts` to select only the fields that are being compared, similar to how `normalize_opts/1` does it below.

Reply:
Thanks for the explanation :+1: 

I changed the function to call `Keyword.keys` in both select and Map.take